Netwrix/netwrix_auditor/ingest/parser.yml

```
@@ -0,0 +1,87 @@
+name: netwrix_auditor_parser
+pipeline:
+  - name: cef_parser
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: general-setter
+    description: ''
+    filter: ''
+    external: null
+  - name: user-email-setter
+    description: ''
+    filter: '{{ "@" in cef_parser.message.suser }}'
+    external: null
+  - name: user-name-domain-setter
+    description: ''
+    filter: '{{ "\\" in cef_parser.message.suser }}'
+    external: null
+  - name: user-only-setter
+    description: ''
+    filter: '{{ "\\" not in cef_parser.message.suser and "@" not in cef_parser.message.suser }}'
+  - name: document-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.cat == "Document" }}'
+    external: null
+  - name: logon-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Logon Activity" }}'
+    external: null
+  - name: exchange-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Exchange Online" }}'
+    external: null
+stages:
+  general-setter:
+    actions:
+      - set:
+          event.kind: '{{ cef_parser.message.DeviceProduct }}'
+          event.category: '{{ cef_parser.message.DeviceEventClassID |lower }}'
```

`event.code` is more suitable for this information
```suggestion
          event.code: '{{ cef_parser.message.DeviceEventClassID |lower }}'
```
<br/>

`event.category` has restricted values (see [ECS specification](https://www.elastic.co/guide/en/ecs/8.4/ecs-allowed-values-event-category.html#ecs-event-category-configuration)). I suggest for `event.category` to use `['configuration']` and for `event.type`, to use `['change']` according the tests.